WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -1734,11 +1734,15 @@ static GSourceFuncs _handlerIntervention =
 
             // WebInspector
             if (_config.Inspector.Value().empty() == false) {
+#ifdef WEBKIT_GLIB_API
+                Core::SystemInfo::SetEnvironment(_T("WEBKIT_INSPECTOR_HTTP_SERVER"), _config.Inspector.Value(), !environmentOverride);
+#else
                 if (_config.Automation.Value()) {
                     Core::SystemInfo::SetEnvironment(_T("WEBKIT_INSPECTOR_SERVER"), _config.Inspector.Value(), !environmentOverride);
```

Comment:
I believe we still want to use WEBKIT_INSPECTOR_SERVER for automation. or have some other option to specify that WEBKIT_INSPECTOR_SERVER  should be set instead of HTTP_SERVER

Reply:
Done.